src/NServiceBus.Persistence.AzureStorage/FodyWeavers.xml

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Weavers>
+<Weavers GenerateXsd="false">
```

Comment:
What does this do?

Reply:
Does not create additional files we don not need.